src/test/java/com/foundation/salesforce/steps/OpportunitySteps.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * @(#) OpportunitySteps.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package com.foundation.salesforce.steps;
+
+import com.foundation.salesforce.core.api.OpportunityApi;
+import com.foundation.salesforce.core.restClient.RestClientApi;
+import com.foundation.salesforce.entities.Context;
+import com.foundation.salesforce.entities.Opportunity;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.When;
+import io.restassured.response.Response;
+
+import java.util.Map;
+
+/**
+ * OpportunitySteps class
+ *
+ * @author Cristian Lujan
```

Comment:
Please put point at the that your name.

Reply:
That is not necessary according to our code convention.